spot-client/src/spot-tv/ui/components/meeting-frame/ZoomMeetingFrame/ZoomMeetingFrame.js

```diff
@@ -0,0 +1,227 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getZoomConfiguration, setSpotTVState } from 'common/app-state';
+import { logger } from 'common/logger';
+import { COMMANDS, SERVICE_UPDATES } from 'common/remote-control';
+import { parseMeetingUrl } from 'common/utils';
+import { events } from 'common/zoom';
+
+import meetingFramePropTypes from '../meetingFramePropTypes';
+
+import ZoomIframeManager from './ZoomIframeManager';
+
+/**
+ * The iFrame used to to display a Zoom meeting.
+ *
+ * @extends React.Component
+ */
+export class ZoomMeetingFrame extends React.Component {
```

Comment:
`JitsiMeetingFrame` contains code which emits analytics for meeting duration and participant count. I guess it's my fault I've put it into component instead of doing through actions. Anyway that's something to be added to the TODO list. Is there a way to get participant count in zoom sdk?